.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: ruby
+rvm:
+ - 2.4
+ - 2.5
+ - 2.6
+ - ruby-head
+matrix:
+  allow_failures:
+    - rvm: ruby-head
```

Comment:
let's add 
```branches:
         only: master
```
So we don't waste a bunch of time building it twice. (as it is doing now)